App/PyImports/Models/StatusModel.py

```diff
@@ -0,0 +1,145 @@
+from PySide2.QtCore import Qt, QObject
+from PySide2.QtGui import QStandardItem, QStandardItemModel
+
+import PyImports.Helpers as Helpers
+from PyImports.StatusObjects import StatusItem, StatusList
+
+class StatusModel(QObject):
+    def __init__(self, calculator, parent=None):
+        super().__init__(parent)
+
+        # major properties
+        self._calculator = calculator
+
+        # Create the status items
+        chiItem = StatusItem('chiSq', title='Goodnes-of-fit (\u03c7\u00b2)', additionalData=1)
```

Comment:
`goodness` rather than `goodnes`. Please correct the UI-exposed occurrences

Reply:
:man_facepalming: